core/state-machine/src/backend.rs

```diff
@@ -329,10 +344,14 @@ impl<H: Hasher> Backend<H> for InMemory<H> {
 		self.inner.get(&None).into_iter().flat_map(|map| map.iter().map(|(k, v)| (k.clone(), v.clone()))).collect()
 	}
 
-	fn keys(&self, prefix: &Vec<u8>) -> Vec<Vec<u8>> {
+	fn keys(&self, prefix: &[u8]) -> Vec<Vec<u8>> {
 		self.inner.get(&None).into_iter().flat_map(|map| map.keys().filter(|k| k.starts_with(prefix)).cloned()).collect()
 	}
 
+	fn child_keys(&self, storage_key: &[u8], prefix: &[u8]) -> Vec<Vec<u8>> {
+		self.inner.get(&Some(storage_key.to_vec())).into_iter().flat_map(|map| map.keys().filter(|k| k.starts_with(prefix)).cloned()).collect()
```

Comment:
This lines exceed the 120 characters limit, can we wrap it, e.g.:

```suggestion
		self.inner.get(&Some(storage_key.to_vec()))
			.into_iter()
			.flat_map(|map| 
				map.keys().filter(|k| 
					k.starts_with(prefix)
				)
				.cloned()
			)
			.collect()
```